### DISCUSSION TITLE
[jbang] [ERROR] Could not download quarkus-cli-2.7.5.Final-runner.jar

### DISCUSSION THREAD

**Discussion by lokize** [quarkusio/quarkus#24429] [Q&A] [answered]
hello friends, on this computer i'm using windows 11 with ubuntu on wsl, every time i try to install quarkus it shows the following problem:

![image](https://user-images.githubusercontent.com/19701630/159170198-76ecd4bd-0ab5-40ef-86a7-fece868f8d55.png)

PS: I already added quarkus to jbang trusted sources.

[a comment by an automated account is not shown]

**Comment by gsmet** [comment]
AFAICS, the file is here: https://repo1.maven.org/maven2/io/quarkus/quarkus-cli/2.7.5.Final/quarkus-cli-2.7.5.Final-runner.jar .

Could it be a proxy problem of some sort? Have you tried the `--verbose` option as mentioned in the error message.

**Comment by lokize** [comment]
yes, link is correct, printscreen in this thread running with --verbose output.

**Reply by maxandersen** [reply]
Do you have any proxies/firewall that could be blocking them?

**Reply by lokize** [reply]
thanks for reply guys, no proxy in this network.

====EDIT====

this problem just happen in wsl...

**Comment by lokize** [comment, marked as answer]
today i removed ubuntu wsl and put centos and it worked perfectly, maybe a problem with ubuntu... Anyway thanks for the help friends!

**Reply by maxandersen** [reply]
okey - let us know if it happens again. WSL can sometimes (especially on older versions) be a bit pick with respect to network.
